src/generator/AutoRest.Go/Model/ParameterGo.cs

```diff
@@ -225,8 +236,8 @@ public static string Validate(this IEnumerable<ParameterGo> parameters, HttpMeth
 
       foreach (var p in parameters)
       {
-        var name = p.SerializedName.Value.IsApiVersion()
-            ? "client." + ParameterGo.ApiVersionName
+        var name = p.IsAPIVersion
+            ? "APIVersion"
```

Comment:
In my latest local version, I actually skip the parameters that return true for IsAPIVersion, so the string const is gone :)